backend/app/notification/__init__.py

```
@@ -0,0 +1 @@
+from .email import *
```

Ruff (0.8.0)

`from .email import *` used; unable to detect undefined names

(F403)